src/request/export/Export.css

```diff
@@ -186,6 +186,15 @@ ul.nq-list li > .nq-icon {
     background-color: rgb( 255, 255, 255, .0); /* transparent white */
 }
 
+.page#login-file-success {
+    opacity: 0;
+    transition: opacity .3s .1s ease;
```

Comment:
Are you sure it's supposed to be `ease`? :D

Reply:
Likely not, but that is why @julianbauer is also a reviewer here ;)

Reply:
For opacity ease is alright :)